gravitee-am-service/src/main/java/io/gravitee/am/service/impl/DomainServiceImpl.java

```diff
@@ -659,6 +659,10 @@ private Completable validateDomain(Domain domain) {
             }
         }
 
+        if(domain.getCorsSettings() != null && domain.getCorsSettings().getAllowedOrigins().isEmpty()) {
```

Comment:
Can `domain.getCorsSettings().getAllowedOrigins()` be ever null ? Can lead to NPE